biomage/experiment/copy.py

```diff
@@ -0,0 +1,33 @@
+import click
+
+from utils.config import get_config
+from utils.data import copy_experiments_to
+
+
+@click.command()
+@click.argument(
+    "experiment_id",
+    required=True,
+)
+@click.argument(
+    "sandbox_id",
+    required=True,
+)
+@click.argument(
+    "origin",
+    default="production",
+)
+@click.argument(
+    "destination",
+    default="staging",
```

Comment:
I am concerned about the potential to shoot ourselves in the foot. There is no check about an exiting experiment being overwritten. I would like to have some confirmation before overwriting something in production.

Reply:
this is an issue permissions that should be enforced by IAM. However, so far there's no need to copy to production and better safe than sorry so I've disabled copying to production.